### ISSUE TITLE
Initial 3 WotG maw pairs not responding to use feather, so cannot be unlocked/used normally

### ISSUE THREAD

**Issue by RAIST5150** [DerpyProjectGroup/topaz#386]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x ] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x ] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x ] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x ] I have specified what branch this happens on branch: Everything/Base branch

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

Discovered on Canaria a couple nights ago and verified Friday night (6/25), running against everything branch.  Kain updated on 6/23 to pull in some new commits, then again this weekend (6/26?), and the problem persists.

Batallia Downs, Rolanberry Fields, and Sauromugue Champaign maws--both past and present.  None of them prompt you to use the white feather to use them.  The secondary maw pairs (Pashhow, Meriph, etc.) behave as normal... prompting you to use the white feather and then port you back and forth just fine.  Abbyssea maws appear to behave as normal as well... seems only these three zones tied to the initial WotG Mission 1 (Cavernous Maws) are impacted.

They work properly to start WotG... porting you to the past and all.  But once the pure white feather KI is rewarded, the past versions of those three zones will not prompt to use the feather so they can be activated in the present.

Likewise, if you already had them unlocked, they do not prompt to use the feather and port you from either the present or the past. They in fact do nothing... not even a "Nothing Happens" message.

The individual lua's for each maw appear to be in synch codewise, so either these specific maw lua's are missing something important or the issue may reside in the globals/maws.lua?

```lua
-----------------------------------
-- Area: Batallia Downs
--  NPC: Cavernous Maw
-- !pos -48 0.1 435 105
-- Teleports Players to Batallia Downs [S]
-----------------------------------
require("scripts/globals/maws")
-----------------------------------
local entity = {}

entity.onTrade = function(player, npc, trade)
end

entity.onTrigger = function(player, npc)
    xi.maws.onTrigger(player, npc)
end

entity.onEventUpdate = function(player, csid, option)
end

entity.onEventFinish = function(player, csid, option)
    xi.maws.onEventFinish(player, csid, option)
end

return entity
```

```lua
-----------------------------------
-- Area: Pashhow Marshlands
--  NPC: Cavernous Maw
-- !pos 418 25 27 109
-- Teleports Players to Pashhow Marshlands [S]
-----------------------------------
require("scripts/globals/maws")
-----------------------------------
local entity = {}

entity.onTrade = function(player, npc, trade)
end

entity.onTrigger = function(player, npc)
    xi.maws.onTrigger(player, npc)
end

entity.onEventUpdate = function(player, csid, option)
end

entity.onEventFinish = function(player, csid, option)
    xi.maws.onEventFinish(player, csid, option)
end

return entity
```

**Comment by claywar** [comment]
This *might* be fixed by #379

**Comment by RAIST5150** [comment]
crap. 

my bad... didn't click the closed PR link.  Meant to post this Saturday morning, but got tied up and was rushing to get this up today since I didn't spot any mentions of a fix on the Discord yet.

Can verify/close this after Kain's next pull.

**Comment by RAIST5150** [comment]
Kain updated Canaria today (included #379), confrmed the maws are working properly again.
